app/src/controllers/plugins/sagas.js

```diff
@@ -0,0 +1,29 @@
+import { all, takeEvery, put } from 'redux-saga/effects';
+import { URLS } from 'common/urls';
+import { fetchDataAction } from 'controllers/fetch';
+import {
+  FETCH_PLUGINS,
+  FETCH_GLOBAL_INTEGRATIONS,
+  NAMESPACE,
+  GLOBAL_INTEGRATIONS_NAMESPACE,
+} from './constants';
+
+function* fetchGlobalIntegrations() {
+  yield put(fetchDataAction(GLOBAL_INTEGRATIONS_NAMESPACE)(URLS.globalIntegrationsByPluginName()));
```

Comment:
For such actions you don't need sagas. You can do it the following way:
```
export const fetchGlobalIntegrationsAction = () =>
  fetchDataAction(GLOBAL_INTEGRATIONS_NAMESPACE)(URLS.globalIntegrationsByPluginName()));
```

Reply:
Oh, thanks. Fixed.